config/config.go

```diff
@@ -174,19 +222,26 @@ func getKubeconfigPaths(homeDir string) []string {
 func GetDefaultCluster(requestIDHeader, activityName, cmdLine, cmdAPIEndpoint string) (clusterID string, err error) {
 	// Go through available orgs and clusters to find all clusters
 	if Config.Token == "" {
-		return "", errors.New("User not logged in.")
+		return "", errors.New("user not logged in")
 	}
-	client := gsclientgen.NewDefaultApiWithBasePath(cmdAPIEndpoint)
+
+	clientConfig := client.Configuration{
+		Endpoint:  cmdAPIEndpoint,
+		Timeout:   10 * time.Second,
+		UserAgent: UserAgent(),
+	}
+	apiClient := client.NewClient(clientConfig)
+
 	authHeader := "giantswarm " + Config.Token
-	orgsResponse, _, err := client.GetUserOrganizations(authHeader, requestIDHeader, activityName, cmdLine)
+	orgsResponse, _, err := apiClient.GetUserOrganizations(authHeader, requestIDHeader, activityName, cmdLine)
 	if err != nil {
 		return "", err
 	}
 	if orgsResponse.StatusCode == 10000 {
```

Comment:
Ah OK I was getting confused between status code and response code. Cool it can be taken care of with the v4 migration.